pkg/rpmdb_testcase.go

```diff
@@ -2785,6 +2785,46 @@ var (
 		{0, "ostree-libs", "2019.6", "2.el8", "x86_64", "ostree-2019.6-2.el8.src.rpm", 981292, "LGPLv2+", "CentOS", "", nil, nil, nil},
 	}
 
+	// docker run --rm -it registry.suse.com/bci/minimal:15.3 bash
+	// rpm -qa --queryformat "\{%{EPOCH}, \"%{NAME}\", \"%{VERSION}\", \"%{RELEASE}\", \"%{ARCH}\"\, \"%{SOURCERPM}\", %{SIZE}, \"%{LICENSE}\", \"%{VENDOR}\", \"\", nil, nil, nil\},\n" | sed "s/(none)/0/g"
+	SLE15_3_bci = []*PackageInfo{
```

Comment:
Can we clarify this case is for NDB? Like SLE15WithNDB or something like that.

Reply:
absolutely, done.